metadata-ingestion/src/datahub/ingestion/graph/client.py

```diff
@@ -1445,6 +1445,51 @@ def run_assertions_for_asset(
 
         return res["runAssertionsForAsset"]
 
+    def get_entities_v2(
+        self,
+        entity_name: str,
+        urns: List[str],
+        aspects: List[Type[Aspect]],
+        with_system_metadata: bool = True,
+    ) -> Optional[Dict]:
```

Comment:
is a raw dict the best return type? is there no stronger type we can use as a response from openapi?

Reply:
The dict is because we return multiple entities and multiple aspects, and the structure is as follows:
```
{ 
  entity_urn: {
    aspect1_key: Aspect1,
    aspect2_key: Aspect2,
 }
```

The actual aspects are parsed into native python structures.